doc/ug/getting_started_fpga.md

```diff
@@ -151,6 +151,8 @@ $ cd $REPO_TOP
 $ /tools/openocd/bin/openocd -s util/openocd -f board/lowrisc-earlgrey-nexysvideo.cfg
 ```
 
+Note that you must use the RISC-V fork of OpenOCD avaliable from: https://github.com/riscv/riscv-openocd. For more instructions on building OpenOCD refer to: https://github.com/lowRISC/opentitan/blob/master/doc/ug/install_instructions/index.md#openocd.
```

Comment:
Thank for this patch Alistair, could I request a few small changes:
* Use one line per sentence, as in the rest of the rile
* It would be more consistent with our general style to put this as "... `use the [RISC-V fork of OpenOCD](https://github.com/riscv/riscv-openocd).`"
* Please integrate the link into the text and use the relref shortcode. e.g. `See the [install instructions]({{< relref "install_instructions#openocd" >}}) for guidance on building OpenOCD.`

Reply:
Done